src/main/java/org/shanerx/tradeshop/commands/Executor.java

```diff
@@ -58,29 +60,48 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
                     return true;
                 }
 
-                String[] help = new String[10];
+                StringBuilder sb = new StringBuilder();
```

Comment:
This is a good one. I don’t know why we made use of arrays, it’s much more efficient this way.

Reply:
we had the array then a loop at the end to add them all to the string builder which makes no sense when you can just use stringbuilder for all of it